hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ContainerImporter.java

```diff
@@ -17,9 +17,13 @@
  */
 package org.apache.hadoop.ozone.container.replication;
 
+import java.util.HashSet;
+import java.util.Set;
```

Comment:
Nit: please add in existing `import java...` lines block.

Reply:
@adoroszlai all changes are updated. Plz recheck